components/bt/host/bluedroid/api/include/api/esp_gap_ble_api.h

```diff
@@ -697,48 +779,58 @@ typedef enum {
 
 /**
   * @brief Structure associated with ESP_AUTH_CMPL_EVT
+  * * The bluetooth low energy authentication complete callback type.
   */
 typedef struct
 {
-    esp_bd_addr_t             bd_addr;               /*!< BD address of peer device */
-    bool                      key_present;           /*!< True if the link key value is valid; false otherwise */
-    esp_link_key              key;                   /*!< Link key associated with peer device */
-    uint8_t                   key_type;              /*!< The type of link key */
-    bool                      success;               /*!< True if authentication succeeded; false otherwise */
-    esp_ble_auth_fail_rsn_t   fail_reason;           /*!< The HCI reason/error code for failure when success is false */
-    esp_ble_addr_type_t       addr_type;             /*!< Peer device address type */
-    esp_bt_dev_type_t         dev_type;              /*!< Device type */
-    esp_ble_auth_req_t        auth_mode;             /*!< Authentication mode */
-} esp_ble_auth_cmpl_t;                               /*!< The ble authentication complete cb type */
-
-/**
-  * @brief union associated with ble security
+    esp_bd_addr_t         bd_addr;               /*!< Basic rate address peer device. */
+    bool                  key_present;           /*!< Valid link key value in key element */
```

Comment:
For the boolean type, please indicate which behavior corresponds to true and which corresponds to false.
For example:
True if xxxxx; false otherwise